lib/export-utils.js

```diff
@@ -29,8 +29,12 @@ class ExportUtils {
             !gitHubPersonalAccessToken ||
             !gitHubPersonalAccessToken.trim().length
         ) {
-            const envGitHubPersonalAccessToken = process.env.CLA_ASSISTANT_UTILS_GITHUB_PERSONAL_ACCESS_TOKEN;
-            if (envGitHubPersonalAccessToken && envGitHubPersonalAccessToken.trim().length) {
+            const envGitHubPersonalAccessToken =
+                process.env.CLA_ASSISTANT_UTILS_GITHUB_PERSONAL_ACCESS_TOKEN;
```

Comment:
This should probably be extracted from the env in the test code and not in the library code - keep the library code clean.